pressbooks/pressbooks-multi-institution
# fix: improve validation error message

This PR addresses https://github.com/pressbooks/pressbooks-multi-institution/issues/31#issuecomment-2010065900 and https://github.com/pressbooks/pressbooks-multi-institution/issues/104#issuecomment-2009653476. 

To test:
1. Visit the assign books and assign users pages
2. Observe that the page URLs have been changed without a change to functionality or appearance
3. Visit the add institution page
4. Enter invalid text in the email domain field.
5. Observe the resulting error message.

Here's what the resulting error messages for invalid email addresses will look like:

![Screenshot from 2024-03-20 12-55-07](https://github.com/pressbooks/pressbooks-multi-institution/assets/13485451/ebd1e88d-8100-4ca1-8af5-41da44b471f5)
![Screenshot from 2024-03-20 12-54-46](https://github.com/pressbooks/pressbooks-multi-institution/assets/13485451/877b0b83-c7be-47ef-8cda-19a58d68e32b)

Here's what a message that also includes an error for an email domain used elsewhere will look like:
![Screenshot from 2024-03-20 13-08-32](https://github.com/pressbooks/pressbooks-multi-institution/assets/13485451/20cbf6e5-9da8-45f6-a08a-9e6c31ae213c)
